packages/browser-wallet/src/popup/pages/AddTokens/AddTokens.scss

```
@@ -1,42 +1,62 @@
-.tokens {
-    &__add {
-        &__container {
-            display: flex;
-            flex-direction: column;
-            width: 100%;
-            height: 100%;
-            padding: rem(15px) rem(10px);
-        }
+$element-border-radius: rem(5px);
 
-        &__element {
-            padding: rem(10px) rem(20px);
-            margin: 0 rem(-10px);
-            display: flex;
-            border-bottom: rem(1px) solid $color-lines;
+.add-tokens {
+    &__container {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        height: 100%;
+        padding: rem(15px) rem(10px);
+    }
 
-            p {
-                margin: 0;
-                margin-left: rem(10px);
-            }
-        }
+    &__element {
+        padding: rem(10px);
+        display: flex;
+        background-color: $color-bg;
 
-        &__input {
-            margin-bottom: rem(10px);
+        &:not(:last-of-type) {
+            border-bottom: rem(1px) solid $color-lines;
         }
 
-        &__add-token {
-            display: flex;
-            justify-content: space-around;
-            margin: 0 rem(-10px) rem(10px);
+        &:first-child {
+            border-top-left-radius: $element-border-radius;
+            border-top-right-radius: $element-border-radius;
         }
 
-        &__submit {
-            margin-top: auto;
+        p {
+            margin: 0;
+            margin-left: rem(10px);
         }
+    }
 
-        &__token-display {
-            max-height: rem(20px);
-            max-width: rem(20px);
-        }
+    &__input {
+        margin-bottom: rem(10px);
+    }
+
+    &__add-token {
+        display: flex;
+        justify-content: space-around;
+        margin: 0 rem(-10px) rem(10px);
```

I think this is where I meant to change something with the comment of something looking nicer:

```suggestion
        margin-bottom: rem(10px);
        
        .form-input {
            flex: 1;
            margin-right: rem(10px);
        }
```